src/main/java/uk/ac/ebi/eva/evaseqcol/service/SeqColExtendedDataService.java

```diff
@@ -47,72 +50,110 @@ public List<SeqColExtendedDataEntity> addAll(List<SeqColExtendedDataEntity> seqC
         return repository.saveAll(seqColExtendedDataList);
     }
 
-    public List<SeqColExtendedDataEntity> getAll() {
-        return repository.findAll();
-    }
-
     /**
-     * Return the 3 extracted seqcol objects (names, lengths and sequences) of the given naming convention*/
-     public List<SeqColExtendedDataEntity> constructLevelTwoSeqCols(AssemblyEntity assemblyEntity, AssemblySequenceEntity sequenceEntity,
-                                                                    SeqColEntity.NamingConvention convention, String accession){
-        // Sorting the chromosomes' list (assemblyEntity) and the sequences' list (sequencesEntity) in the same order
-        sortReportAndSequencesByRefseq(assemblyEntity, sequenceEntity, accession);
-        SeqColExtendedDataEntity namesEntity;
-        SeqColExtendedDataEntity lengthsEntity;
-        SeqColExtendedDataEntity sequencesEntity;
-        JSONExtData jsonNamesObject = new JSONExtData();
-        JSONExtData jsonLengthsObject = new JSONExtData();
-        JSONExtData jsonSequencesObject = new JSONExtData();
-        List<String> sequencesNamesObject = new LinkedList<>(); // Array of sequences' names
-        List<String> sequencesLengthsObject = new LinkedList<>(); // Array of sequences' lengths
-        List<String> sequencesObject = new LinkedList<>(); // // Array of actual sequences
+     * Return the extendedData object for the given digest*/
+    Optional<SeqColExtendedDataEntity> getExtendedAttributeByDigest(String digest) {
```

Comment:
Suggest adding explicit access level (public/private/protected) to the methods....

Reply:
Sure, sorry for that.